templates/proxy/configmap-properties.yaml

```diff
@@ -16,11 +16,81 @@ data:
     # See https://graphdb.ontotext.com/documentation/ for supported properties
     graphdb.connector.port={{ .Values.proxy.containerPorts.http }}
     graphdb.proxy.hosts={{ include "graphdb-proxy.cluster.nodes" . }}
+    {{- with .Values.configuration.tls }}
```

Comment:
I'm not sure what happens if both the proxy and graphdb are configured for SSL, have you tested this scenario?